cmd/web5/cmd_didweb.go

```diff
@@ -12,7 +11,7 @@ type didWebCreate struct {
 	Domain string `arg:"" help:"The domain name for the DID." required:""`
 }
 
-func (c *didWebCreate) Run(_ context.Context) error {
```

Comment:
Also found (per PR feedback) out we don't need these, Kong will dynamically inject based on needs